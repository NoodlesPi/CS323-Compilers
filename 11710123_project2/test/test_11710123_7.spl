int apple(int a){
    return a;
}
float apple(int b){
    return 1.0;
}
int main(){
    int appla[10];
    // type 10
    int app = appla[10][5];
    // type 2
    peach(5);
    return 1;
}