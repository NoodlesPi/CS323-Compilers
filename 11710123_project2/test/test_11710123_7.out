Error type 4 at Line 4: function is redefined
Error type 10 at Line 10: applying indexing operator ([...]) on non-array type variables
Error type 5 at Line 10: unmatching type
Error type 2 at Line 12: function is invoked without definition
